themes/default/content/docs/reference/pulumi-yaml.md

```diff
@@ -8,39 +8,74 @@ menu:
-        - `binary`: applies to Go, .NET and Java projects.
-            - **Go**: A string that specifies the name of a pre-built executable to look for on your path. If not specified, go sources in $CWD will be invoked via `go run`.
-            - **.NET**: A string that specifies the path of a pre-built .NET assembly. If not specified, a .NET project in $CWD will be invoked via `dotnet run`.
-            - **Java**: A string that specifies the path of a pre-built jar file.
-        - `virtualenv`: applies to Python projects only. A string that specifies the path to a virtual environment to use when running the program. New Python projects created with `pulumi new` have this option set by default. If not specified, Pulumi will invoke the `python3` command it finds on $PATH (falling back to `python`) to run the Python program. If you'd like to use a virtual environment without the `virtualenv` option, you'll need to run any `pulumi` commands (such as `pulumi up`) from an activated virtual environment shell (or, if using a tool like [Pipenv](https://github.com/pypa/pipenv), prefix any `pulumi` commands with `pipenv run pulumi ...`).
-        - `compiler`: applies to Pulumi YAML projects only. A executable and arguments that will emit to standard out a Pulumi YAML program to run.
+## Attributes
 
-- `description`: (optional) a friendly description about your project.
+| Name | Required | Description | 
+| - | - | - | - |
+| `name` | required | The name of the project containing alphanumberic characters, hyphens, underscores, and period. |
+| `runtime` | required | The installed language runtime of the project: `nodejs`, `python`, `go`, `dotnet`, `java` or `yaml` to use. | 
+| `description` | optional | The description of the project. |
+| `main` | optional | Overide of the working directity default of the location of the Pulumi program. |
+| `stackConfigDir` | optional | Config directory location relative to the location of `Pulumi.yaml` | 
+| `backend` | optional | The [backend]({{< relref "/docs/intro/concepts/state" >}}) of the project. |
+| `options` | optional | Additional project options. |
+| `template` | optional | Config to be used when creatinig new stacks in the project. |
+| `plugins` | optional | Override the plugin selection. |
 
-- `main`: (optional) an override for the main program's location. By default, the program's working directory is assumed to be the location of `Pulumi.yaml`. To choose a different location, use the `main` property. For example, if your Pulumi program is in a subdirectory `infra/`, use `main: infra/`.
+### `runtime` options
 
-- `stackConfigDir`: (optional) directory to store stack-specific configuration files, relative to location of `Pulumi.yaml`.
+The runtime attribute has an additional optional `options` option where you can specify additional runtime configuration.
```

Comment:
i know `optional options option` is ridic, what can i use instead?

Reply:
How about:

> The runtime attribute has an additional property called `options` where you can further specify runtime configuration.